modules/comments/components/PollCommentItem.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```

Comment:
just a wrapper on top of the generic comment item to extract the twitter message and some logic regarding to the poll vote